tests/extract_outline_test.py

```diff
@@ -0,0 +1,198 @@
+doi:10.1080/1941126X.2011.601225. 
+
+2. “Sci-Hub: Removing Barriers in the Way of Science.” Accessed September 6, 2016. http:// 
+
+sci-hub.bz/. 
+
+3. “Elsevier Inc. et al. v. Sci-Hub et al. “Docket Item 50 | United States Courts Archive.” 
+September 15, 2015. https://www.unitedstatescourts.org/federal/nysd/442951/50-0.html. 
+4. Bohannon, John. “Who’s Downloading Pirated Papers? Everyone.” Science 352, no. 6285 
+
+(April 29, 2016): 508–512. doi:10.1126/science.352.6285.508. 
+
+5. Bohannon, John. “The Frustrated Science Student behind Sci-Hub.” Science 352, no. 6285 
+
+(April 29, 2016): 511. doi:10.1126/science.352.6285.511. 
+
+6. Cabanac, Guillaume. “Bibliogifts in LibGen? A Study of a Text-Sharing Platform Driven 
+by Biblioleaks and Crowdsourcing.” Journal of the Association for Information Science and 
+Technology 67, no. 4 (March 27, 2015): 874–884. doi:10.1002/asi.23445. 
+
+7. “Does Sci-Hub Phish for Credentials?” Sauropod Vertebra Picture of the Week. February 
+
+25, 2016. https://svpow.com/2016/02/25/does-sci-hub-phish-for-credentials/. 
+
+8. Banks, Marcus. “Sci-Hub: What It Is and Why It Matters.” American Libraries Magazine, 
+May 31, 2016. https://americanlibrariesmagazine.org/2016/05/31/why-sci-hub-matters/. 
+9. Smith, David. “Sci-Hub: How Does It Work?” The Scholarly Kitchen. February 25, 2016. 
+
+https:/scholarlykitchen.sspnet.org/2016/02/25/sci-hub-how-does-it-work/. 
+
+10. Waddell, Kaveh. “The Research Pirates of the Dark Web.” The Atlantic, February 9, 2016. 
+http://www.theatlantic.com/technology/archive/2016/02/the-research-pirates-of-the-dark- 
+web/461829/. 
+
+11. Association of American Publishers. “AAP Statement on Sci-Hub.” May 10, 2016. http:// 
+
+newsroom.publishers.org/aap-statement-on-sci-hub. 
+
+12. Library of Congress, Copyright Office. “International Copyright Relations of the United 
+
+States.”  Accessed October 20, 2016. http://purl.access.gpo.gov/GPO/LPS101162. 
+
+13. Bendezú-Quispe, Guido, Wendy Nieto-Gutiérrez, Josmel Pacheco-Mendoza, and Alvaro 
+Taype-Rondan. “Sci-Hub and Medical Practice: An Ethical Dilemma in Peru.” The Lancet 
+Global Health 4, no. 9 (September, 2016): e608. doi:10.1016/S2214-109X(16)30188-7. 
+
+14. Dobbs, David. “Testify: The Open-Science Movement Catches Fire.” WIRED. January 30, 
+
+2012. https://www.wired.com/2012/01/testify-the-open-science-movement-catches-fire/. 
+
+15. McNutt, Marcia. “My Love-Hate of Sci-Hub.” Science 352, no. 6285 (April 29, 2016): 497. 
+
+doi:10.1126/science.aaf9419. 
+
+16. Zipf, George Kingsley. Human Behavior and the Principle of Least Effort: An Introduction 
+
+to Human Ecology. Cambridge, MA: Addison-Wesley Press, 1949. 
+
+17. Ruff, Corinne. “Librarians Find Themselves Caught Between Journal Pirates and 
+Publishers.” February 18, 2016. The Chronicle of Higher Education. http://www.chronicle. 
+com/article/Librarians-Find-Themselves/235353/. 
+
+View publication stats
+View publication stats
+
+"""
+
+fname = "samples/sci.pdf"
```

Comment:
Constants should be written in capital case (see [PEP-8 spec](https://peps.python.org/pep-0008/#constants)). Also, it would be useful to have a short description about what the constant is used for.